src/status_im/contexts/wallet/collectible/tabs/about/view.cljs

```diff
@@ -0,0 +1,47 @@
+(ns status-im.contexts.wallet.collectible.tabs.about.view
+  (:require [quo.core :as quo]
+            [quo.theme]
+            [react-native.core :as rn]
+            [status-im.contexts.wallet.collectible.tabs.about.style :as style]
+            [utils.re-frame :as rf]))
+
+(def link-cards
+  [{:title               "BAYC"
+    :icon                :social/link
+    :address             "boredapeyachtclub"
+    :customization-color :social/link
+    :on-press            #(js/alert "pressed")}
+   {:title               "Twitter"
+    :icon                :social/twitter
+    :address             "@BoredApeYC"
+    :customization-color :social/twitter
+    :on-press            #(js/alert "pressed")}
+   {:title               "Opensea"
+    :icon                :social/opensea
+    :address             "Bored Ape Yacht Club"
+    :customization-color :social/opensea
+    :on-press            #(js/alert "pressed")}])
+
+(defn- view-internal
+  []
+  (let [window-width (rf/sub [:dimensions/window-width])
+        item-width   (- (/ window-width 2) 28)]
```

Comment:
what is 28? can you put that in a constant so we know what it is?

Reply:
Done